specifications/M1/verifiable.presentation.protocol.md

```diff
@@ -171,7 +171,40 @@ the `credentials` endpoint
 
 `POST /credentials`
 
-If successful, credentials will be created or updated and a `200 OK` returned.
+If the POST is successful, credentials will be created and a `201 Created` returned.
+
+The POST body is a JSON object with the following properties:
+
+- `@context`: REQUIRED. Specifies a valid [Json-Ld context](https://www.w3.org/TR/json-ld11/#the-context).
+- `@type`: REQUIRED. A string specifying the credential type.
+- `credentials`: REQUIRED. A Json structure corresponding to the schema
+  specified [below](#the-credential-object).
+
+The following is a non-normative example of the JSON body:
+
+```json
+{
+  "@context": [
+    "https://w3id.org/tractusx-trust/v0.8"
+  ],
+  "@type": "Credentials",
+  "credentials": [
+    {
+      "format": "",
+      "payload": ""
+    }
+  ]
+}
+```
+
+## The `Credential` Object
+
+The credentials property contains a single Json `Credential` object or an Array of `Credential` objects.
```

Comment:
can we make it so that it _always_ contains an array, even if there's just one object? This makes implementing DTOs for this a lot easier:
```suggestion
The `credentials` property contains an array of `credential` objects.
```

Reply:
Sure, we can force this with Json-Ld, so OK